packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -114,8 +114,8 @@ export default class Store extends EventEmitter<{
   _bridge: FrontendBridge;
 
   // Computed whenever _errorsAndWarnings Map changes.
-  _cachedErrorCount: number = 0;
-  _cachedWarningCount: number = 0;
+  _cachedUniqueErrorCount: number = 0;
+  _cachedUniqueWarningCount: number = 0;
```

Comment:
Nit on naming: "unique" isn't really accurate, i.e. in the example from PR desc, every error can still be different, even if they originate from the same component.

If I understand the PR example correctly, it's a "count of components with any [errors/warnings]".

Maybe we can pick a concise version of the following?

```suggestion
  _cachedComponentWithAnyErrorCount: number = 0;
  _cachedComponentWIthAnyWarningCount: number = 0;
```

Reply:
Yeah, that makes more sense, let me change that